src/main/java/seedu/address/model/project/Task.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.model.project;
+
+import java.time.LocalDate;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+
+public class Task {
+
+    // Identity fields
+    private final Name taskName;
+
+    private Person person;
+
+    private boolean status;
+    private Integer progressCounter = 0;
+
+    private LocalDate deadlineDate;
+
+    private Integer priorityNumber;
```

Comment:
Instead of this perhaps an "urgent" status or "important" status would be better?